Tests/SonarScanner.MSBuild.PreProcessor.Test/PreProcessorTests.cs

```diff
@@ -250,15 +250,15 @@ public async Task Execute_NoPlugin_ReturnsFalseAndLogsError()
     }
 
     [TestMethod]
-    public async Task Execute_NoProject_ReturnsTrue()
+    public async Task Execute_NoQualityProfile_ReturnsTrue()
     {
         using var context = new Context(TestContext, new MockObjectFactory(false));
         context.Factory.Server.DownloadQualityProfile(null, null, null).ReturnsForAnyArgs((string)null);
 
         (await context.Execute()).Should().BeTrue();
 
         context.AssertDirectoriesCreated();
-        context.AssertDownloadMethodsCalled(1, 1, 2, 0); // no quality profile assigned to project
+        context.AssertDownloadMethodsCalled(properties: 1, allLanguages: 1, qualityProfile: 2, rules: 0);   // no quality profile assigned to project
```

Comment:
I still don't understand the test and the assertion. `Execute_NoProject_ReturnsTrue` indicated that there was somehow "No project". The assertion indicates that two qualityProfiles were downloaded. What is the point here and why were there quality profiles added before. This all makes no sense to me.